pylint_django/tests/input/func_model_no_explicit_unicode_str_compat.py

```diff
@@ -0,0 +1,19 @@
+"""
+Ensures that django models with a __str__ method defined in an ancestor and
+python_2_unicode_compatible decorator applied are flagged correctly as not
+having explicitly defined __unicode__
+"""
+#  pylint: disable=missing-docstring
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class BaseModel(models.Model):
+    def __str__(self):
+        return 'Foo'
+
+
+class SomeModel(BaseModel):
```

Comment:
Add this class to `func_noerror_unicode_py2_compatible.py` which already includes the rest.

For the record test names usually start with `func_noerror`.

Reply:
This code should actually emit `model-no-explicit-unicode`. How do I check for that?

Reply:
See *.txt files in the test suite for clues. You have to annotate the lines in the Python source with the expected pylint error, e.g. `# [model-no-explicit-unicode]` and then inside the txt file you describe the error in more details which are `error-key:line_number:class name or empty:error string from pylint (1st line only)`